### PR TITLE
Moved reverseproxy configmap to aws/base

### LINKED ISSUE #79: Research moving the reverseproxy config into base.
Need to look into if the reverseproxy configmap can be moved to base for a single point of editing.  This would help with #73

### PR DESCRIPTION
**Pull request by jhujasonw**
Fixes #79
All 4 namespaces pull from the same reverseproxy-configmap.yaml now which is in aws/base.

### CODE REVIEW

**Comment by jhujasonw** [comment]
The conflict seems to be that the reverseproxy-configmap.yaml was updated in main and deleted in this change.  Will merge once it's approved via commandline.

**Comment by jhujasonw** [comment]
Merging will also close #73 for all namespaces and will close #77 for all namespaces.

**Review by derekbelrose — APPROVED** [review]
This all looks good.  Thank you for simplifying things!

**Pull request merged**
